agr_java_core/src/main/java/org/alliancegenome/es/index/site/cache/IndexerCache.java

```diff
@@ -142,5 +144,110 @@ public void addCachedFields(SearchableItemDocument document) {
 
 
     }
+    public void addCachedAlleleFields(Iterable<AlleleVariantSequence> documents) {
+        for (AlleleVariantSequence document : documents) {
+            addCachedAlleleFields(document);
+        }
+    }
+    public void addCachedAlleleFields(AlleleVariantSequence document) {
```

Comment:
This needs to not be a copy and paste of `addCachedFields`

Reply:
That's true, but this would be quick fix